common/src/main/scala/net/psforever/packet/game/WeaponDelayFireMessage.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2016 PSForever.net to present
+package net.psforever.packet.game
+
+import net.psforever.packet.{GamePacketOpcode, Marshallable, PacketHelpers, PlanetSideGamePacket}
+import scodec.Codec
+import scodec.codecs._
+
+/** WeaponDelayFireMessage seems to be sent when a weapon has a delayed projectile after firing, such as the knife.
+  *
+  * See PlayerStateMessageUpstream for explanation of seq_time.
```

Comment:
You can use `[[PlayerStateMessageUpstream]]` to make a doc reference (link in HTML)